CMakeLists.txt

```diff
@@ -51,6 +51,16 @@ else()
   message(STATUS "Configuring Shaderc to avoid building examples.")
 endif()
 
+option(SHADERC_SKIP_EXECUTABLES "Skip building executables." ${SHADERC_SKIP_EXECUTABLES})
+if(NOT ${SHADERC_SKIP_EXECUTABLES})
+  set(SHADERC_ENABLE_EXECUTABLES ON)
+endif()
+if (${SHADERC_SKIP_EXECUTABLES})
```

Comment:
I think this flag is backwards? If it's set to skip, it prints it's building.

Reply:
If I'm reading this right, if `SHADERC_SKIP_EXECUTABLES` is `true`, meaning we'll skip the creation of executables, the message printed is `Configuring Shaderc to build executables` which, seems like the opposite of what we've done, no?

Reply:
Right, this condition should be `${SHADERC_ENABLE_EXECUTABLES}`